requirements/requirements.txt

```diff
@@ -1,6 +1,7 @@
 ovos-backend-client~=0.0.6
 ovos-plugin-manager>=0.0.23a20
-ovos-microphone-plugin-alsa>=0.0.0a3
-ovos-utils~=0.0.32
+ovos-microphone-plugin-alsa~=0.0.0
+ovos-utils~=0.0.32,>=0.0.33a12
+ovos-config~=0.0.9,>=0.0.10a2
```

Comment:
might want to bump min version to include https://github.com/OpenVoiceOS/ovos-config/pull/52